src/pages/public_registration/LogPanel.tsx

```diff
@@ -13,7 +13,7 @@ interface LogPanelProps {
 export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
   const { t } = useTranslation();
   return (
-    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '0.5rem' }}>
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', mt: '0.5rem', bgcolor: 'secondary.main' }}>
```

Comment:
Merkelig felt øverst her, som jeg antar vi ikke ønsker?

![bilde](https://github.com/user-attachments/assets/5468b8a6-7d85-4115-bba4-88561acb69d3)

```suggestion
    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', bgcolor: 'secondary.main' }}>
```